### PR TITLE
Created 404 Page

### LINKED ISSUE #20: [404] Create a 404 page
Taking inspiration from: https://online.codingblocks.com/404 create default nextjs 404 page.

### PR DESCRIPTION
**Pull request by prathambatra**
fixes #20 
Signed-off-by: prathambatra <batrapratham999@gmail.com>

### CODE REVIEW

**Comment by prathambatra** [comment]
Screenshot from 2020-05-17 22-17-22
Screenshot from 2020-05-17 22-17-33

**Comment by prathambatra** [comment]
@hereisnaman please review.

**Comment by 0xnmn** [comment]
No need to render navbar and sidebar

**Comment by 0xnmn** [comment]
Center align the component with col-8 width

**Comment by prathambatra** [comment]
okay doing changes

**Comment by prathambatra** [comment]
@hereisnaman I have done the required changes.

**Comment by prathambatra** [comment]
@hereisnaman please check.

File: app/components/PageNotFound/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './style.scss';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+const PageNotFound = () => {
+  return (
+    <div className="col-8 col-centre text-center">
```
Comment (0xnmn) [inline app/components/PageNotFound/index.js]: No col-center required. use offset classes. Let it be col-12 and col-md-8. Also move the col classes to the custom404 component.

File: app/components/PageNotFound/style.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+.heading {
+  font-size: 8rem;
+  padding-top: 4rem;
+}
+
+.col-centre {
```
Comment (0xnmn) [inline app/components/PageNotFound/style.scss]: not required.

File: app/pages/404.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PageNotFound from 'Components/PageNotFound';
+
+const Custom404 = () => {
+  return (
+    <div>
+      <PageNotFound />
+    </div>
```
Comment (0xnmn) [inline app/pages/404.js]: Wrap whole thing inside a '.row' and more the col classes here.

**Review by 0xnmn — CHANGES_REQUESTED** [review]

**Comment by prathambatra** [comment]
@hereisnaman  done

**Comment by prathambatra** [comment]
@hereisnaman have a look.

**Comment by 0xnmn** [comment]
> @hereisnaman have a look.

Can you add updated screnshots?

**Comment by prathambatra** [comment]
> > @hereisnaman have a look.
> 
> Can you add updated screnshots?

Screenshot from 2020-05-18 01-01-43

**Review by 0xnmn — APPROVED** [review]

**Pull request merged**